@xen-orchestra/lite/.prettierrc.cjs

```diff
@@ -1,15 +1 @@
-module.exports = {
-  importOrder: [
-    "^[^/]+$",
-    "<THIRD_PARTY_MODULES>",
-    "^@/components/(.*)$",
-    "^@/composables/(.*)$",
-    "^@/libs/(.*)$",
-    "^@/router/(.*)$",
-    "^@/stores/(.*)$",
-    "^@/views/(.*)$",
-  ],
-  importOrderSeparation: false,
-  importOrderSortSpecifiers: true,
-  importOrderParserPlugins: ["typescript", "decorators-legacy"],
-};
+module.exports = {};
```

Comment:
Remove this file if no longer necessary?

Reply:
If we remove the file, then the root config is used and the whole XO Lite project would need to be updated (quotes, comma etc.) so for now I kept the file.

Reply:
I think we should add a comment in this file to explain this.